### PR TITLE
Minor worker fixes

### PR DESCRIPTION
**Pull request by subdavis**
* Turns out heartbeat was useful. I'm learning lots of stuff about celery.  I was trying to test the behavior more robustly by running a worker and turning my VPN on/off, pulling out my ethernet cable, etc.  Without the heartbeat, the rabbit server doesn't know when a connection has been lost.  Mailbox reservation gets stuck and messages start to flow into a mailbox for a worker that is no longer listening.  With heartbeat, the mailbox will eventually be cleared and the job with get rescheduled on another worker.
* Add `KWIVER_DEFAULT_LOG_LEVEL`.  Some processes don't respect this variable, but that's not dive's concern.
* I've experimentally found that the timeout error handling works well, so I've dropped the timeout to a more reasonable value.

Our cancellation error handling means that mingle isn't needed.  Without mingle, you get scary errors about invalid state transitions, but girder enforces job state integrity and the revoked job gets removed from the queue after it errors. Gossip doesn't apply to our deployment configuration at all.

### CODE REVIEW

File: server/dive_tasks/celeryconfig.py [hunk 1]
```diff
@@ -44,12 +44,11 @@
 if broker_url is None:
     raise RuntimeError('CELERY_BROKER_URL must be set')
 
-broker_heartbeat = False
 worker_send_task_events = False
 # https://docs.celeryproject.org/en/stable/userguide/configuration.html#std-setting-worker_prefetch_multiplier
 worker_prefetch_multiplier = 1
 # https://docs.celeryproject.org/en/v4.4.6/userguide/configuration.html#broker-connection-timeout
-broker_connection_timeout = 12
+broker_connection_timeout = 6
```
Comment (BryonLewis) [inline server/dive_tasks/celeryconfig.py:51]: what are the downsides to this being more lenient?  From the documentation it is the worker connection timeout so I'm assuming that means the connection from the individual running the worker to the message broker.  Just thinking of edge cases, Hawaii, or satellite internet from a boat?

**Review by BryonLewis — COMMENTED** [review]
Just a general question about the timeout.

File: server/dive_tasks/celeryconfig.py (hunk 1 above)
Comment (subdavis) [reply server/dive_tasks/celeryconfig.py:51]: User experience, really.  12 seconds is a long time.  If nothing happens for that long when you should be getting a job (or a message about having lost connection) you might get confused or frustrated.

That said, maybe it's not a big deal?

**Review by BryonLewis — APPROVED** [review]
Looks good.

**Pull request merged**